.github/workflows/build-and-save-contract.yml

```diff
@@ -0,0 +1,56 @@
+name: Build and save DDC SC binaries
+on: [push]
```

Comment:
Can we add also for releases and pre-releases? It will help if we decide to make release, but last "push" artifact expired in github storage.